<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttendSettings</class>
 <widget class="QDialog" name="AttendSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Attendance settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>List 1</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>3</number>
        </property>
        <item>
         <widget class="QListWidget" name="listWidget_1st">
          <property name="locale">
           <locale language="English" country="UnitedKingdom"/>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::InternalMove</enum>
          </property>
          <property name="movement">
           <enum>QListView::Free</enum>
          </property>
          <property name="currentRow">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="2">
           <widget class="QToolButton" name="removeButton_1st">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../pics.qrc">
              <normaloff>:/pics/list-remove-32x32.png</normaloff>:/pics/list-remove-32x32.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QToolButton" name="addButton_1st">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../pics.qrc">
              <normaloff>:/pics/list-add-32x32.png</normaloff>:/pics/list-add-32x32.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>List 2</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QListWidget" name="listWidget_2nd">
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::InternalMove</enum>
          </property>
          <property name="movement">
           <enum>QListView::Free</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="2">
           <widget class="QToolButton" name="removeButton_2nd">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../pics.qrc">
              <normaloff>:/pics/list-remove-32x32.png</normaloff>:/pics/list-remove-32x32.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QToolButton" name="addButton_2nd">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../pics.qrc">
              <normaloff>:/pics/list-add-32x32.png</normaloff>:/pics/list-add-32x32.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>List 3</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QListWidget" name="listWidget_3rd">
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::InternalMove</enum>
          </property>
          <property name="movement">
           <enum>QListView::Free</enum>
          </property>
          <property name="spacing">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="2">
           <widget class="QToolButton" name="removeButton_3rd">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../pics.qrc">
              <normaloff>:/pics/list-remove-32x32.png</normaloff>:/pics/list-remove-32x32.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QToolButton" name="addButton_3rd">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../pics.qrc">
              <normaloff>:/pics/list-add-32x32.png</normaloff>:/pics/list-add-32x32.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Others</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLineEdit" name="spinBox_1_labelEdit">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maxLength">
           <number>50</number>
          </property>
          <property name="placeholderText">
           <string>Option title</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Warning : modify orders changes the Attendance manager lists orders </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pics.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AttendSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AttendSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
